rocks-db/Cargo.toml

```diff
@@ -41,6 +41,7 @@ usecase = { path = "../usecase" }
 tempfile = { workspace = true }
 bubblegum-batch-sdk = { git = "https://github.com/metaplex-foundation/bubblegum-batch-sdk.git", rev = "0d529f5" }
 num-traits = { workspace = true }
+flatbuffers = { version="24.3.25", features = ["serialize"]}
```

Comment:
This version is used as the generated code seems to be compatible with it, but not with 23... used in the project. It's either the code may be regenerated with 23, or existing updated to 24, which I'm not aware will be easy/possible.